tests/integration/test_default_model_selection_integration.py

```
@@ -0,0 +1,150 @@
+    assert default_model.is_default is True
+
+    e2 = await llm_config_manager.get_embedding_model("local-embed")
+    assert e2.is_default is False
+
+
+@pytest.mark.asyncio
+async def test_set_nonexistent_default_raises_error(llm_config_manager):
+    """Test setting a non-existent model as default raises error"""
+    with pytest.raises(
+        Exception
+    ):  # storage likely raises a ValueError or similar, manager re-raises or wraps
```

Ruff (0.14.14)

[warning] Do not assert blind exception: `Exception`

(B017)